packages/astro/package.json

```diff
@@ -0,0 +1,25 @@
+{
+	"name": "@unpic/astro",
+	"version": "0.0.1",
+	"type": "module",
+	"license": "MIT",
+	"exports": {
+		".": "./index.ts"
+	},
+	"files": [
+		"src",
+		"index.ts"
+	],
+	"keywords": [
+		"astro"
+	],
+	"devDependencies": {
+		"astro": "^2.0.12"
+	},
+	"dependencies": {
+		"@unpic/core": "workspace:^"
+	},
+	"peerDependencies": {
+		"astro": "^2.0.0-beta.0"
```

Comment:
Is there a reason the peer deps and dev deps are different?

Reply:
I left it unchanged after generating the folder. As far as I can tell there is no reason. I'll set both versions to " ^2.0.0"